servers/cromwell/README.md

```diff
@@ -15,15 +15,22 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
     export CROMWELL_URL=http://192.168.0.106:8000/api/workflows/v1
     ```
 
-2. Add a file named `config.json` to `job-manager/servers/cromwell/jobs` that contains the username and password for the specified cromwell instance (You may want to skip this step if the Cromwell instance does not have HTTP Basic Authentication):
+2. (Optional) By default, the shim layer uses **5** `sync` [Gunicorn workers](http://docs.gunicorn.org/en/stable/settings.html#worker-class), the number of workers is **5**. 
```

Comment:
Why 5 workers, is that just arbitrary?

Reply:
Oh, so according to the documentation(http://docs.gunicorn.org/en/stable/settings.html#workers), Gunicorn suggests a range of `2-4 x $(NUM_CORES)` for the number of workers to get the best load. People always use `2N+1` as a practice. The default number 5 here assumes you are using at least 2-core CPU (since even 4-core so so common nowadays), so `2 x 2 + 1 = 5`